src/Security/Factory/OidcFactory.php

```diff
@@ -60,6 +61,7 @@ public function createAuthenticator(
         ->addArgument($config['check_path'])
         ->addArgument($config['login_path'])
         ->addArgument($config['user_identifier_property'])
+        ->addArgument($config['user_identifier_from_idtoken'])
         ->addArgument($config['enable_remember_me']);
```

Comment:
```suggestion
        ->addArgument($config['enable_remember_me'])
        ->addArgument($config['user_identifier_from_idtoken']);
```

For BC reasons of the class.